app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -222,7 +230,9 @@ class OnboardingFragmentTest {
   @Test
   fun testOnboardingFragment_checkSlide1Description_isCorrect() {
     launch(OnboardingActivity::class.java).use {
-      onView(withId(R.id.onboarding_slide_view_pager)).perform(scrollToPage(1))
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(isRoot()).perform(orientationPortrait())
       onView(
```

Comment:
I just wanted to check if this could work keeping the above-mentioned points in mind.
After scrolling action, changing the orientation to landscape, again changing back to portrait, and then checking the assertion.
This worked but one test case is still falling - 
![Screenshot from 2021-01-21 12-22-50](https://user-images.githubusercontent.com/54636525/105315435-51073f80-5be5-11eb-9880-31e87bff6a25.png)

Reply:
I don't think we should do this just to pass the pass. 

All the steps in the test we perform are something we can think of that a user will also perform and based on that the test is passing or not. After a swipe, a user will change orientation two times as this is not a default way. 

To check what's going wrong with the test, we need to look at how the lifecycle is working when the test is getting performed. 

Other point, as mentioned earlier, if the view is in all the pages then we need to first test that the view on the specific page is completely visible or not, and then we can match the text or image.

Reply:
@anandwana001 Yes, as mentioned in [this](https://github.com/oppia/oppia-android/pull/2482#pullrequestreview-573033619) comment, I'll revert these changes. 
@BenHenning suggested that I should investigate more on this, so I just wanted to share the results after my investigation.